src/main/java/org/opensearch/ad/indices/AnomalyDetectionIndices.java

```diff
@@ -605,33 +617,32 @@ public void initAnomalyResultIndexDirectly(
      *
      * @param actionListener action called after create index
      */
-    // @anomaly-detection.create-detector Commented this code until we have support of Job Scheduler for extensibility
-    // public void initAnomalyDetectorJobIndex(ActionListener<CreateIndexResponse> actionListener) {
-    // try {
-    // CreateIndexRequest request = new CreateIndexRequest(".opendistro-anomaly-detector-jobs")
-    // .mapping(getAnomalyDetectorJobMappings(), XContentType.JSON);
-    // request
-    // .settings(
-    // Settings
-    // .builder()
-    // // AD job index is small. 1 primary shard is enough
-    // .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
-    // // Job scheduler puts both primary and replica shards in the
-    // // hash ring. Auto-expand the number of replicas based on the
-    // // number of data nodes (up to 20) in the cluster so that each node can
-    // // become a coordinating node. This is useful when customers
-    // // scale out their cluster so that we can do adaptive scaling
-    // // accordingly.
-    // // At least 1 replica for fail-over.
-    // .put(IndexMetadata.SETTING_AUTO_EXPAND_REPLICAS, minJobIndexReplicas + "-" + maxJobIndexReplicas)
-    // .put("index.hidden", true)
-    // );
-    // adminClient.indices().create(request, markMappingUpToDate(ADIndex.JOB, actionListener));
-    // } catch (IOException e) {
-    // logger.error("Fail to init AD job index", e);
-    // actionListener.onFailure(e);
-    // }
-    // }
+    public void initAnomalyDetectorJobIndex(ActionListener<CreateIndexResponse> actionListener) {
+        try {
+            CreateIndexRequest request = new CreateIndexRequest(AnomalyDetectorJob.ANOMALY_DETECTOR_JOB_INDEX)
+                .mapping(getAnomalyDetectorJobMappings(), XContentType.JSON);
+            request
+                .settings(
+                    Settings
+                        .builder()
+                        // AD job index is small. 1 primary shard is enough
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        // Job scheduler puts both primary and replica shards in the
+                        // hash ring. Auto-expand the number of replicas based on the
+                        // number of data nodes (up to 20) in the cluster so that each node can
+                        // become a coordinating node. This is useful when customers
+                        // scale out their cluster so that we can do adaptive scaling
+                        // accordingly.
+                        // At least 1 replica for fail-over.
+                        .put(IndexMetadata.SETTING_AUTO_EXPAND_REPLICAS, minJobIndexReplicas + "-" + maxJobIndexReplicas)
+                        .put("index.hidden", true)
+                );
+            client.indices().create(request, markMappingUpToDate(ADIndex.JOB, actionListener));
```

Comment:
I guess we do have AdminClient in SDKRestClient. @dbwiddis ?

Reply:
Yeah it's just a self-reference to the same client, but lets us avoid changing the diff.  In other classes I've assigned the `adminClient` parameter to `client.admin()` which is really just `client` but keeps the rest of the code the same. We should probably do so here.